src/app/modules/item/components/permissions-edit-dialog/permissions-edit-dialog.component.html

```diff
@@ -33,6 +33,13 @@
 
       <div class="dialog-container" *ngIf="state.isReady">
         <div class="dialog-content">
+          <ng-container *ngIf="item.string.title">
+            <p class="dialog-description" i18n>
+              These are the permissions given to "{{ permReceiverName }}" on "{{ item.string.title }}"
+              <ng-container *ngIf="group.isUser && permGiverName">by "{{ permGiverName }}"</ng-container>
+            </p>
+          </ng-container>
```

Comment:
Sorry, my bad, could you change 
"These are the permissions given to "Armelle Bonenfant (arbonenfant)" on "Blockly Basic Task" by "Super group of mine (Thomas ®)"" 
to 
"These are the permissions given by "Super group of mine (Thomas ®)" to "Armelle Bonenfant (arbonenfant)" on "Blockly Basic Task"" 
?

Reply:
Fixed